src/pages/build-apps/guides/transaction-signing.md

```diff
@@ -27,38 +27,58 @@ See the public registry tutorial for a concrete example of this functionality in
 
 ## Install dependency
 
+~> With the recent launch of the Stacks Blockchain, the production release of `@stacks/connect` does not include a few key changes to help you build apps that integrate with the Stacks Mainnet and Testnet. If you're following this guide, you should install a beta version of `@stacks/connect` with the `launch-misc` tag on NPM, as described below. We expect this code to be officially released into the `@stacks/connect` package soon.
+
 The following dependency must be installed:
 
 ```
-npm install @stacks/connect
+npm install @stacks/connect@launch-misc
 ```
 
 ## Initiate session
 
-Users must authenticate to an app before the `connect` package will work to prompt them for signing and broadcasting transactions to the Stacks blockchain with an authenticator such as [the Stacks Wallet](https://blockstack.org/wallet).
+Users must authenticate to an app before the `connect` package will work to prompt them for signing and broadcasting transactions to the Stacks blockchain with an authenticator such as [the Stacks Wallet](https://www.hiro.so/wallet/install-web).
 
 See the authentication guide before proceeding to integrate the following transaction signing capabilities in cases where `userSession.isUserSignedIn()` returns `true`.
 
 [@page-reference]
 | /build-apps/guides/authentication
 
+## Get the user's Stacks address
+
+After your user has authenticated with their Stacks Wallet, you can get their Stacks address from their `profile`.
+
+```ts
+const profile = userSession.loadUserData().profile.stxAddress;
+
+const mainnetAddress = stxAddresses.mainnet;
+// "SP2K5SJNTB6YP3VCTCBE8G35WZBPVN6TDMDJ96QAH"
+const testnetAddress = stxAddresses.testnet;
+// "ST2K5SJNTB6YP3VCTCBE8G35WZBPVN6TDMFEVESR6"
+```
+
 ## Prompt to transfer STX
 
 Call the `openSTXTransfer` function provided by the `connect` package to trigger the display of a transaction signing prompt for transferring STX:
 
 ```tsx
 import { openSTXTransfer } from '@stacks/connect';
+import { StacksTestnet } from '@stacks/network';
+
+const network = new StacksTestnet();
```

Comment:
would it make sense to leave a comment suggesting how to change this up for mainnet?

Reply:
I added a section at the bottom and links to it in each reference table, but if you'd prefer to be more explicit we can!

Reply:
yeah, I'd err on the side of things that could go wrong, will. can already see how devs wouldn't read to the end of the page and be puzzled.. but that's an optimization for sure. your call